src/metatrain/experimental/nanopet/schema-hypers.json

```diff
@@ -27,6 +27,16 @@
         "num_gnn_layers": {
           "type": "integer"
         },
+        "heads": {
+          "type": "object",
+          "patternProperties": {
+            ".*": {
+              "type": "string",
+              "enum": ["linear", "mlp"]
+            }
+          },
+          "additionalProperties": false
+        },
```

Comment:
Is this feature documented to the user anywhere?

Reply:
Added it, also for SOAP-BPNN